lnbits/core/views/generic.py

```diff
@@ -387,6 +387,44 @@ async def manifest(usr: str):
     }
 
 
+@core_html_routes.get("/node", response_class=HTMLResponse)
+async def node(request: Request, user: User = Depends(check_admin)):
+    if not settings.lnbits_node_ui:
+        raise HTTPException(status_code=HTTPStatus.NOT_FOUND)
```

Comment:
- maybe `HTTP 503 Service Unavailable` is more appropriate
- `404 Not Found` is if the route/resource does not exist